components/function-controller/pkg/apis/serverless/v1alpha2/function_types.go

```diff
@@ -284,4 +283,17 @@ func (c *Condition) IsTrue() bool {
 	return c.Status == v1.ConditionTrue
 }
 
-func (f *Function) Hub() {}
```

Comment:
It was remaining from trying to use the spoke/hub pattern.. we didn't go that way because we had to merge two CRs together and it wasn't possible to extend it to use a client.

Reply:
ok, thanks for the explanation